_posts/2016/2016-02-17-development-from-the-treches.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "Development from the 'southern' trenches"
```

Comment:
I'm not sure about the title - maybe you could amend it a bit so that it "attracts" also Google ;-).